packages/apollo-server-plugin-operation-registry/src/__tests__/agent.test.ts

```diff
@@ -7,19 +7,16 @@ import {
   genericStorageSecret,
   nockStorageSecret,
   nockBase,
-  nockLegacyGoodManifest,
-  genericSchemaHash,
-  hashedServiceId,
   nockGoodManifestsUnderStorageSecret,
   defaultStore,
   defaultTestAgentPollSeconds,
   nockStorageSecretOperationManifest,
   genericApiKeyHash,
-  genericLegacyOperationManifestUrl,
 } from './helpers.test-helpers';
 import Agent, { AgentOptions } from "../agent";
-import { Operation, OperationManifest } from "../ApolloServerPluginOperationRegistry";
-import { fakeTestBaseUrl, getLegacyOperationManifestUrl, getStoreKey, getOperationManifestUrl, urlOperationManifestBase } from "../common";
+import { Operation } from "../ApolloServerPluginOperationRegistry";
+import { fakeTestBaseUrl, getStoreKey, getOperationManifestUrl, urlOperationManifestBase } from "../common";
+import { Logger } from "loglevel";
```

Comment:
Should we consider using this type elsewhere as well going forward? Currently we own this one, which I've used extensively in the past:
https://github.com/apollographql/apollo-server/blob/a926b7eedbb87abab2ec70fb03d71743985cb18d/packages/apollo-server-types/src/index.ts#L143-L149